datasets/ro/onpcsb_sanctions/crawler.py

```diff
@@ -0,0 +1,88 @@
+import csv
+from typing import Dict
+from zavod import Context, helpers as h
+from typing import List
+
+
+DATE_FORMATS = ["%m/%d/%Y", "%Y"]
+
+
+def parse_date(text: str) -> List[str]:
+    # add a function to handle multiple entires
+    if not text:
+        return []
+    date = h.parse_date(text, DATE_FORMATS)
+    return date
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    full_name = row.pop("name")
+    other_name = row.pop("other name")
+    birth_date_1 = parse_date(row.pop("date of birth"))
+    birth_date_2 = parse_date(row.pop("date of birth 2"))
+    birth_place = row.pop("place of birth")
+    nationality = row.pop("nationality")
+    passport_number = row.pop("passport no.")
+    position = row.pop("position")
+    po_box = row.pop("postal code")
+    fiscal_code = row.pop("fiscal code")
+    phone_number = row.pop("phone number")
+    address_1 = row.pop("address_1")
+    address_2 = row.pop("address_2")
+    city = row.pop("city")
+    country = row.pop("country")
+    address = h.make_address(
+        context,
+        full=address_1,
+        remarks=address_2,
+        po_box=po_box,
+        city=city,
+        country=country,
+    )
+    entity_type = row.pop("type")
+    raw_data = row.pop("parsed data")
+
+    entity = None
+    if entity_type == "Person":
+        entity = context.make("Person")
+        entity.id = context.make_id(raw_data)
```

Comment:
I'd suggest using name, date and place of birth here (all of them), and for company, something like name and a key part of the address like the post code.

The benefit of that, is if we get the same data in future in another form, e.g. they add some entities and publish a nice excel file or something, or even a new PDF, the ID hash is more likely to stay the same, meaning dedupe we do on these entities (with other datasets) continues to apply.